server/conn.go

```diff
@@ -789,9 +792,18 @@ func (cc *clientConn) Run(ctx context.Context) {
 		}
 
 		startTime := time.Now()
+
+		ctx := ctx
+		var handle minitrace.TraceHandle
+		traceEnabled := tidbtrace.Enable.Load()
+		if traceEnabled {
+			ctx, handle = minitrace.StartRootSpan(ctx, cc.packetToSQL(data), rand.Uint64(), &tidbtrace.Context{})
```

Comment:
@c4pt0r You can replace `rand.Uint64()` with another meaningful trace ID here.

Reply:
Sure